internal/config/user_test.go

```diff
@@ -18,6 +18,14 @@ func TestAddUser(t *testing.T) {
 	})
 }
 
+func TestUserExists2(t *testing.T) {
+	context := Context{
+		ContextDetails: ContextDetails{},
+		Name:           "",
+	}
+	UserExists(context)
```

Comment:
>UserExists(context) [](http://example.com/codeflow?start=1&length=19)

shouldn't this validate it returns false?

Reply:
and how about a variant where it returns true?

Reply:
Agree.  Added both.